sys-block/parted/parted-3.2.recipe

```diff
@@ -102,13 +102,12 @@ defineDebugInfoPackage parted$secondaryArchSuffix \
 BUILD()
 {
 	autoreconf
+	CPPFLAGS="-Wimplicit-fallthrough=0" \
+	LDFLAGS="-lblkid -luuid -lintl" \
```

Comment:
```diff
-	LDFLAGS="-lblkid -luuid -lintl" \
+	LIBS="-lblkid -luuid -lintl" \
```

This still doesn't seem right however, the build system should correctly link these libraries as needed without having us to append it here.

Reply:
Oops, I forgot to fix this.

Reply:
Can you check if it's possible to drop some `LIBS`? After all, they should not be necessary. The build system should detect whether they should be linked or not.

Reply:
Good catch. (It looks like we don't need to specify `LIBS`.) Thanks!
I guess I added it because it was not working without it, but maybe the following change is what made it work without the `LIBS`:
```diff
-	cmd:gettext
+	cmd:gettext$secondaryArchSuffix
```